Source/Scene/DracoLoader.js

```diff
@@ -119,8 +119,13 @@ define([
             return;
         }
 
+        loadResources.activeDecodingTasks++;
         loadResources.primitivesToDecode.dequeue();
-        return promise.then(addDecodededBuffers(taskData, model, context));
+        return promise.then(addDecodededBuffers(model, context))
```

Comment:
Can we rename `addDecodededBuffers` to `createAddDecodedBuffersCallback` or something similar, otherwise it looks like a bug in promise usage in the code.

Although why is this even a separate function?  Might be more readable to just declare it inline since it's the only place it's used.